tools/sidekick/config.json

```diff
@@ -3,6 +3,7 @@
   "host": "business.adobe.com",
   "previewHost": "main--bacom--adobecom.aem.page",
   "liveHost": "main--bacom--adobecom.aem.live",
+  "trustedHosts": ["milo.adobe.com"],
```

Comment:
Given [the context](https://github.com/adobecom/da-bacom-blog/pull/47#discussion_r2058947061) from the other PR, it seems like we should add `*--bacom--adobecom.aem.page` to account for the bulk publishing, loc, etc tools.

@narcis-radu Is that correct?

Reply:
> it seems like we should add *--bacom--adobecom.aem.page to account for the bulk publishing, loc, etc tools.

The current site is always a trusted host (host, preview host, live host), it's just which other hosts are allowed to as well.